### PR TITLE
Fix: Eth/Usd historical prices

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Change source for eth/usd historical price to get a precise value 

Fixes ENG-939

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

**Comment by KMKoushik** [comment]
@DemolaJames Build failed? Can you fix it and push the code?

**Comment by DemolaJames** [comment]
> @DemolaJames Build failed? Can you fix it and push the code?

@KMKoushik issue fixed...you can review now

File: packages/frontend/src/hooks/useUserCrabV2TxHistory.ts [hunk 1]
```diff
@@ -44,12 +47,29 @@ export const useUserCrabV2TxHistory = (user: string, isDescending?: boolean) =>
       },
     },
   )
+ 
+  //get all timestamps found in the user's history once
+  useEffect(() => {
+
+    let timestampsArr : any[] = []
```
Comment (KMKoushik) [inline packages/frontend/src/hooks/useUserCrabV2TxHistory.ts:54]: Why push inside map
timestampArr = userTxes.map(tx => tx.timestamp * 1000)

**Review by KMKoushik — APPROVED** [review]

File: packages/frontend/src/hooks/useUserCrabV2TxHistory.ts (hunk 1 above)
Comment (DemolaJames) [reply packages/frontend/src/hooks/useUserCrabV2TxHistory.ts:54]: True ...will fix

**Review by sanandnarayan — APPROVED** [review]
lgtm;

might want to keep an eye on twelvedata limits

**Pull request merged**